apps/web/src/api/hooks/notification-templates/useTemplateFetcher.ts

```diff
@@ -0,0 +1,26 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query';
+import { INotificationTemplate } from '@novu/shared';
+
+import { getTemplateById } from '../../notification-templates';
+import { QueryKeys } from '../../query.keys';
+
+export function useTemplateFetcher(
```

Comment:
improved this hook a little so now it will be possible to pass query options
also moved to the shared API hooks folder